tools/pubchem/pubchem_build.xml

```diff
@@ -0,0 +1,36 @@
+<tool id="pubchem_sparql" name="Pubchem SPARQL Query" version="0.1">
+    <requirements>
+        <requirement type="package" version="1">tar</requirement>
+        <requirement type="package" version="7.1.2">click</requirement>
+        <requirement type="package" version="0.5.0">fastparquet</requirement>
+        <requirement type="package" version="3.1.1">openbabel</requirement>
+        <requirement type="package" version="1.2.1">pandas</requirement>
+        <requirement type="package" version="4.0.30">pyodbc</requirement>
+        <requirement type="package" version="0.5.4">python-snappy</requirement>
+        <requirement type="package" version="3.8">python</requirement>
+    </requirements>
+
+    <command detect_errors="aggressive"><![CDATA[
+        tar -xvf ${pubchem_rdf}
+
+        python '${__tool_directory__}/pubchem_build.py'
```

Comment:
Galaxy treats this as one command, hence the error `tar: unrecognized option '--workers' Try 'tar --help' or 'tar --usage' for more information.` when trying to run this. We could chain it with some logical operators like `&&`, extract to an external script, etc.